apps/fortran/airfoil/airfoil_arg_ptrs/dp/airfoil.F90

```diff
@@ -59,30 +75,33 @@ program AIRFOIL
 
   print *, ncell
   ! allocate sets (cannot allocate in subroutine in F90)
-  allocate ( cell ( 4 * ncell ) )
-  allocate ( edge ( 2 * nedge ) )
-  allocate ( ecell ( 2 * nedge ) )
-  allocate ( bedge ( 2 * nbedge ) )
-  allocate ( becell ( nbedge ) )
-  allocate ( bound ( nbedge ) )
-
-  allocate ( x ( 2 * nnode ) )
-  allocate ( q ( 4 * ncell ) )
-  allocate ( qold ( 4 * ncell ) )
-  allocate ( res ( 4 * ncell ) )
-  allocate ( adt ( ncell ) )
-
-  !allocated simply to test op_fetch_data_idx()
-  allocate ( q_part ( 4 * ncell ) )
+  call op_malloc(ptr_cell, 4 * ncell  * 4)
```

Comment:
OPlus has a method named op_malloc, though with 1 integer*4 argument. Might clash...

Reply:
In Hydra tests did not clash. But yes in the future it could clash. how about op_mem_alloc ( .. )